<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModpackPage</class>
 <widget class="QWidget" name="ModpackPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>781</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="settingsTabs">
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Modpack</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="1">
        <widget class="QLineEdit" name="coordinateEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="repositoryCheck">
         <property name="text">
          <string>Repository</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="platformComboBox"/>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="modpackCheck">
         <property name="text">
          <string>Instance is based on a modpack</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="platformLabel">
         <property name="text">
          <string>Modpack Platform</string>
         </property>
         <property name="buddy">
          <cstring>platformComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="coordinateCheck">
         <property name="text">
          <string>Coordinate</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="repositoryEdit"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="javaTab">
      <attribute name="title">
       <string>Java</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="memoryGroupBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Memor&amp;y</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QCheckBox" name="minMemCheck">
            <property name="text">
             <string>Minimum memory allocation:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="optMemCheck">
            <property name="text">
             <string>Optimal memory allocation:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="minMemSpinBox">
            <property name="toolTip">
             <string>The amount of memory Minecraft is started with.</string>
            </property>
            <property name="suffix">
             <string notr="true"> MB</string>
            </property>
            <property name="minimum">
             <number>128</number>
            </property>
            <property name="maximum">
             <number>65536</number>
            </property>
            <property name="singleStep">
             <number>128</number>
            </property>
            <property name="value">
             <number>1024</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="maxMemCheck">
            <property name="text">
             <string>Maximum memory allocation:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="maxMemSpinBox">
            <property name="toolTip">
             <string>The maximum amount of memory Minecraft is allowed to use.</string>
            </property>
            <property name="suffix">
             <string notr="true"> MB</string>
            </property>
            <property name="minimum">
             <number>128</number>
            </property>
            <property name="maximum">
             <number>65536</number>
            </property>
            <property name="singleStep">
             <number>128</number>
            </property>
            <property name="value">
             <number>1024</number>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="optMemSpinBox">
            <property name="toolTip">
             <string>The amount of memory available to store loaded Java classes.</string>
            </property>
            <property name="suffix">
             <string notr="true"> MB</string>
            </property>
            <property name="minimum">
             <number>64</number>
            </property>
            <property name="maximum">
             <number>999999999</number>
            </property>
            <property name="singleStep">
             <number>8</number>
            </property>
            <property name="value">
             <number>1024</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="javaArgumentsGroupBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Java argumen&amp;ts</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="1" column="1">
           <widget class="QPlainTextEdit" name="jvmArgsTextBox"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="activateUIButton">
     <property name="text">
      <string>Click here to unlock</string>
     </property>
     <property name="description">
      <string>These are the modpack defaults and shouldn't be changed unless you are working on the modpack or want to change its source or target version, or work on it.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>activateUIButton</tabstop>
  <tabstop>settingsTabs</tabstop>
  <tabstop>modpackCheck</tabstop>
  <tabstop>platformComboBox</tabstop>
  <tabstop>repositoryCheck</tabstop>
  <tabstop>repositoryEdit</tabstop>
  <tabstop>coordinateCheck</tabstop>
  <tabstop>coordinateEdit</tabstop>
  <tabstop>minMemCheck</tabstop>
  <tabstop>minMemSpinBox</tabstop>
  <tabstop>maxMemCheck</tabstop>
  <tabstop>maxMemSpinBox</tabstop>
  <tabstop>optMemCheck</tabstop>
  <tabstop>optMemSpinBox</tabstop>
  <tabstop>javaArgumentsGroupBox</tabstop>
  <tabstop>jvmArgsTextBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
